Guides/Global/AllFinished.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+import { Text, View } from 'react-native';
+
+export default class AllFinished extends Component {
```

Comment:
Maybe consider renaming this to be `FinishedGuide` or `GuideFinished`? I think `AllFinished` is a little ambiguous, especially if we don't end up using `All Finished` as a copywriting for that slide.

Reply:
Actually, I think it needs to be `SessionFinished`